src/components/MessagesList/MessagesGroup/Message/Message.spec.js

```diff
@@ -337,6 +339,14 @@ describe('Message.vue', () => {
 				return messageEl
 			}
 
+			beforeEach(() => {
+				useMessageInfoSpy = jest.spyOn(useMessageInfoModule, 'useMessageInfo')
+			})
+
+			afterEach(() => {
+				useMessageInfoSpy.mockRestore()
+			})
```

Comment:
It will be more boilerplace, as it needs to mock return for all props from composable to every suite, at least 
`TypeError: Cannot destructure property 'isEditable' of '(0 , _useMessageInfo.useMessageInfo)(...)' as it is undefined.`